packages/testlab/src/__tests__/integration/docker.integration.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {DockerClient} from '../..';
+import {expect} from '../../expect';
+
+describe('docker', function() {
+  let client: DockerClient;
+
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(30000);
+
+  before(async () => {
+    client = new DockerClient();
```

Comment:
How is this going to run on machines that don't have Docker daemon running? For example Windows build machines like AppVeyor? Personally, I am not running the Docker daemon unless I really need it. I would consider it as a degradation of developer experience if `npm test` in loopback-next started to require a running Docker server.

Reply:
Travis CI builds on MacOS are failing too.

https://travis-ci.com/strongloop/loopback-next/jobs/245526182
```
 1) docker
       "before all" hook for "inspects an image by name":
     Error: connect ENOENT /var/run/docker.sock
      at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1128:14)
```